widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Bubble.kt

```diff
@@ -0,0 +1,52 @@
+@file:Suppress("FunctionName")
+
+package com.glia.widgets.view.unifiedui.theme.defaulttheme
+
+import com.glia.widgets.view.unifiedui.exstensions.composeIfAtLeastOneNotNull
+import com.glia.widgets.view.unifiedui.theme.ColorPallet
+import com.glia.widgets.view.unifiedui.theme.base.BadgeTheme
+import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
+import com.glia.widgets.view.unifiedui.theme.bubble.BubbleTheme
+import com.glia.widgets.view.unifiedui.theme.chat.UserImageTheme
+
+/**
+ * Default theme for Unread messages indicator
+ */
+internal fun BubbleDefaultTheme(pallet: ColorPallet): BubbleTheme? =
+    pallet.run {
+        val badge = BadgeDefaultTheme(this)
+        val userImage = UserImageDefaultTheme(this)
+        composeIfAtLeastOneNotNull(badge, userImage) {
+            BubbleTheme(
+                userImage = userImage,
+                badge = badge,
+            )
+        }
+    }
+
+/**
+ * Default theme for User image
+ */
+internal fun UserImageDefaultTheme(pallet: ColorPallet): UserImageTheme? = pallet.run {
+    composeIfAtLeastOneNotNull(primaryColorTheme, baseLightColorTheme) {
+        UserImageTheme(
+            placeholderColor = baseLightColorTheme,
+            placeholderBackgroundColor = primaryColorTheme,
+            imageBackgroundColor = primaryColorTheme
+        )
+    }
+}
+
+/**
+ * Default theme for User image
+ */
```

Comment:
It seems to me that this comment should say something like "Default theme for badge", shouldn't it?